probes/requiresCodeOwnersReview/impl.go

```diff
@@ -50,9 +77,6 @@ func Run(raw *checker.RawResults) ([]finding.Finding, string, error) {
 		case !*reqOwnerReviews:
 			text = fmt.Sprintf("codeowners review is not required on branch '%s'", *branch.Name)
 			outcome = finding.OutcomeNegative
-		case len(r.CodeownersFiles) == 0:
-			text = "codeowners review is required - but no codeowners file found in repo"
-			outcome = finding.OutcomeNegative
```

Comment:
out of curiosity, what was there a reason for the change?

At first glance the current implementation seems reasonable?

Reply:
In short: To determine when to log at info level here: 

https://github.com/ossf/scorecard/blob/fdf3fb23652cb69e19204421c0cede904bd16061/checks/evaluation/branch_protection.go#L412

The relevant evaluation is here: 

https://github.com/ossf/scorecard/blob/fdf3fb23652cb69e19204421c0cede904bd16061/checks/evaluation/branch_protection.go#L409-L421

`*branch.BranchProtectionRule.RequiredPullRequestReviews.RequireCodeOwnerReviews` can be true while `codeOwnerFiles` can be 0. 

If the probe returns `OutcomeNegative` if `len(codeownersFiles)` is 0, then the evaluation won't know whether to log the `info()` call.

Reply:
Mentioned this to Laurent offline, and he agreed with `OutcomeNegative`. We lose the info statement, but it simplifies remediation later.

Reply:
https://github.com/ossf/scorecard/pull/3759/commits/db37bc1aee27aca1b0aa8ab59e82f73e1970b174